torchtune/models/llama3/_tokenizer.py

```diff
@@ -181,17 +234,18 @@ def tokenize_messages(
         # bos and eos are always masked
         mask = [True]
         for message in messages:
-            tokenized_message = self.tokenize_message(
-                message, tokenize_header=tokenize_header
-            )
+            tokenized_message = self.tokenize_message(message)
```

Comment:
So we don't actually pass tokenize_header through anymore? Is this just because it's always true for llama3 anyways?

Reply:
yeah, kind of an artifact from previous approaches I've tried. I left it in there in case it may be useful in the future. but I'm ok to remove